lib/gui/app/pages/main/controllers/image-selection.js

```diff
@@ -76,18 +77,30 @@ module.exports = function (
       })
 
       osDialog.showError(invalidImageError)
-      analytics.logEvent('Invalid image', image)
+      analytics.logEvent('Invalid image', {
+        image,
+        applicationSessionUuid: store.getState().toJS().applicationSessionUuid,
+        flashingWorkflowUuid: store.getState().toJS().flashingWorkflowUuid
+      })
       return
     }
 
     Bluebird.try(() => {
       let message = null
 
       if (supportedFormats.looksLikeWindowsImage(image.path)) {
-        analytics.logEvent('Possibly Windows image', image)
+        analytics.logEvent('Possibly Windows image', {
+          image,
```

Comment:
This will need to be merged; i.e. `Object.assign({ applicationSessionUuid, flashingWorkflowUuid }, image)` as this would otherwise change the data structure received by mixpanel

Reply:
Yeah true, I didn't think about that. The image has a very different structure, but I'll just merge it with new data for now, we'll be able to change it later.

Reply:
How's this `image`'s structure different from the one before?

Reply:
@jhermsmeier Sorry, I meant it's different from a classic event, so there's gonna be properties of the image on the same level as the various tracking UUIDs. It's not a problem, just something I thought could be improved.